packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -98,15 +126,16 @@
                     </div>
                 </Tile>
 
-                <Tile>
-                    <div class="flex flex-col space-y-2 items-center justify-center w-full">
-                        <!-- TODO: Replace this with the actual staked amount -->
-                        <Text type={TextType.h3}>0i</Text>
-                        <Text color="gray-600" fontWeight={FontWeight.medium} fontSize="12" type={TextType.p}
-                            >{localize('views.accountManagement.details.staked')}</Text
-                        >
-                    </div>
-                </Tile>
+                {#if hasStakingFeature}
+                    <Tile>
+                        <div class="flex flex-col space-y-2 items-center justify-center w-full">
+                            <Text type={TextType.h3}>{getStakedAmount()}</Text>
```

Comment:
I would not use a function here for `getStakedAmount` but rather a reactive variable that updates when the output updates, we will need 2 new reactive variables, eg `rawStakedAmount` & `formattedStakedAmount`